### ISSUE TITLE
Feature request: compact article list

### ISSUE THREAD

**Issue by stewf** [Ranchero-Software/NetNewsWire#941]
One of my favorite things about NNW 3 is the single-line listings with title, source, date, and creator in columns. I don’t expect that same level of customization, but an alternate “compact” view without smaller favicons (or no favicons) and fewer lines per article would be nice.

**Comment by 2stepbay** [comment]
RSS ideally provides a quick scan of article headlines without having to read a lot of text. The ability to limit the description line to 1, 2, 3 etc. provides the user with better options that match their scanning style. Hope to see such a feature in NNW5 soon!

**Comment by 2stepbay** [comment]
Example from how I set up NNW 3
<img width="1322" alt="Screen Shot 2019-09-03 at 16 11 53" src="https://user-images.githubusercontent.com/54873323/64220578-93e83c00-ce65-11e9-89d1-d28cede75a7e.png">

**Comment by noahkuhn** [comment]
+1 Agree. I want this exact same functionality added back in. And actually, also have the option to have my titles across full width with the content below. Same functionality as NNW 3

**Comment by sbarex** [comment]
yes, as it is now it seems to me to be dispersive to read and I cannot get used to it

**Comment by 2stepbay** [comment]
This ought to be an easy change. Surprised it hasn't been made.

**Comment by brentsimmons** [comment]
While wiring up a simple table view is pretty easy, no feature is as easy as just that.

And, critically, there are other things with higher priority at the moment, such as the iOS app, Feedly syncing, content extraction, and some other things.

We’ll get to this — it’s important, for sure — but it has to wait for some other things first.

**Comment by sbarex** [comment]
Probably the readability would be better using for the date label a smaller font with normal weight.

**Comment by sbarex** [comment]
I tried to create a pull request with date aligned on right side: #1247

**Comment by brentsimmons** [comment]
Duplicate of #102
